internal/controller/thanosreceive_controller_test.go

```diff
@@ -306,15 +305,15 @@ config:
         "tenant_matcher_type": "exact",
         "endpoints": [
             {
-                "address": "some-hostname-b.test-resource-test-hashring.treceive.svc.cluster.local:19291",
+                "address": "some-hostname-b.test-resource-test-hashring.treceive.svc.cluster.local:10901",
```

Comment:
Why did port change here?

Reply:
because i had it wrong the first time :D